marlowe-dashboard-client/src/ContractHome/State.purs

```diff
@@ -100,7 +100,7 @@ filledContract1 (Slot currentSlot) =
         , (Role "Seller") /\ Nothing
         ]
 
-    mContract = toCore $ fillTemplate templateContent Escrow.extendedContract
+    mContract = toCore $ fillTemplate templateContent Escrow.fixedTimeoutContract
```

Comment:
We can remove the `slotContent` from the `templateContent` above now. But there's no need - I'm about to change how the dummy contracts are provided in my next PR, and they'll be gone entirely soon enough (they're just for testing until everything's working in the PAB).

Reply:
Yes, I was counting on the instantiation mechanism just ignoring the extra info, and this way it is simpler (because I can use it in Marlowe Playground directly) and easier to revert in the future.